bokeh/models/widgets/buttons.py

```diff
@@ -26,7 +26,7 @@ class AbstractButton(Widget):
     An optional image appearing to the left of button's text.
     """)
 
-    type = Enum(ButtonType, help="""
+    button_type = Enum(ButtonType, help="""
```

Comment:
This is a breaking API change, we need to be sure to document it in the release notes